src/components/document/Header/UserProfile.jsx

```diff
@@ -0,0 +1,12 @@
+import ProfileIcon from "@/assets/pfp.svg"
+
+const UserProfile = () => {
+    // This is a fill in component.
+    // TODO: fill out this functionality (with contexts, whatever) once we have login/authentication
```

Comment:
is there a way to link up Linear so it adds TODOs auto? (I've added link to this comment in linear task)

Reply:
I have no idea actuallyl lol not that I know of